execute.c

```diff
@@ -451,6 +451,40 @@ ssh_command_tty(char *host_name, char *socket_path, Label *host_label, int http_
 	return ret;
 }
 
+int
+scp_archive(char *host_name, char *socket_path, Label *host_label, int http_port, int direction) {
+	int i;
+	int argc;
+	char scp_opt[PLN_LABEL_SIZE];
+	char scp_src[PLN_LABEL_SIZE];
+	char scp_dst[PLN_LABEL_SIZE];
+	char *path;
+	char *argv[32];
+	int ret = 0;
+
+	snprintf(scp_opt, sizeof(scp_opt), "ControlPath=%s", socket_path);
+	argc = append(argv, 0, "scp", "-o", scp_opt, NULL);
```

Comment:
I think, it's worth adding in the man, that archive/restore will work only for files. If one defines a directory, it will fail. And that it should be only one file.
I also curious, what would happen if a file defined in this way in a route file and also set -A or -R?

Reply:
Actually any number of files can be specified

```
_archive: var_mail.tar home.tar
        tar -Pp -cf var_mail.tar /var/mail
        tar -Pp -cf home.tar /home
```

`-A` and `-R` together is probably useless but works:
1. Existing archive files are uploaded (or an error if it does not exist)
2. New archive files are downloaded